catroid/gradle/release_fastlane_tasks.gradle

```diff
@@ -65,7 +65,7 @@ android.productFlavors.all { flavor ->
 
         String fastlaneCommand = "fastlane screengrab" +
                 " --app_package_name 'org.catrobat.catroid'" +
-                " --use_tests_in_packages 'org.catrobat.catroid.screenshots.${flavor.name}'" +
+                " --use_tests_in_packages 'org.catrobat.catroid.screenshots.${flavor.name.toLowerCase()}'" +
```

Comment:
why is to lowerCase just needed for this argument and not the others?

Reply:
This refers to a package name, which are all lowercase by convention. All the other arguments here refer to files and paths, which are camelCase. 
'flavor.name' itself is in camelCase as well. So it is only need for the argument containing the package name.